### PR TITLE
ETCD Service NodePort

### PR DESCRIPTION
**Pull request by rastislavszabo**

### CODE REVIEW

File: k8s/contiv-vpp.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 # This installs the contiv-etcd (ETCD server to be used by Contiv) on the master node in a Kubernetes cluster.
 # In odrer to dump the content of ETCD, you can use kubectl exec command similar to this:
```
Comment (jmedved) [inline k8s/contiv-vpp.yaml]: This installs contiv-etcd (the ETCD server to be used by Contiv) on the master node in a Kubernetes cluster.
  # In order to dump the content of ETCD, you can use the kubectl exec command similar to this:

File: k8s/contiv-vpp.yaml [hunk 2]
```diff
@@ -160,21 +183,21 @@ spec:
...
+        - name: vcl-ldpreload
+          hostPath:
+            path: /usr/libexec/ldpreload/vcl-ldpreload
+        # Used by VPP to communicate with LD_PRELOAD client, later it'll be replaced by socket file directory mount.
         - name: shm
           hostPath:
             path: /dev/shm
-        # VPP binaries, used to test with socket_test_client/server tool
+        # VPP binaries, used to test with socket_test_client/server tool.
         - name: vpp
```
Comment (jmedved) [inline k8s/contiv-vpp.yaml]: VPP binaries, used to test with the socket_test_client/server tool.

File: k8s/contiv-vpp.yaml [hunk 3]
```diff
@@ -160,21 +183,21 @@ spec:
         - name: cni-net-dir
           hostPath:
             path: /etc/cni/net.d
-        # Used by VPP to communicate with LD_PRELOAD client, later it'll be replaced by socket
+        # LD_PRELOAD library
+        - name: vcl-ldpreload
+          hostPath:
+            path: /usr/libexec/ldpreload/vcl-ldpreload
+        # Used by VPP to communicate with LD_PRELOAD client, later it'll be replaced by socket file directory mount.
```
Comment (jmedved) [inline k8s/contiv-vpp.yaml]: Used by VPP to communicate with LD_PRELOAD clients, later it'll be replaced by a socket file directory mount.

**Comment by rastislavszabo** [comment]
grammar fixed, will be part of one of my next MR.

**Pull request merged**